src/components/Collapsible/Collapsible.tsx

```diff
@@ -30,165 +24,58 @@ export interface CollapsibleProps {
   children?: React.ReactNode;
 }
 
-type AnimationState =
-  | 'idle'
-  | 'measuring'
-  | 'closingStart'
-  | 'closing'
-  | 'openingStart'
-  | 'opening';
-
-interface State {
-  height?: number | null;
-  animationState: AnimationState;
-  open: boolean;
-}
-
-const ParentCollapsibleExpandingContext = createContext(false);
```

Comment:
This is needed for nested collapsibles
```
import React, {useState, useCallback} from 'react';

import {Page, Card, Stack, Button, Collapsible, TextContainer} from '../src';

export function Playground() {
  return (
    <Page title="Playground">
      <CollapsibleExample />
    </Page>
  );
}

function CollapsibleExample() {
  const [open, setOpen] = useState(true);
  const [innerOpen, setInnerOpen] = useState(true);

  const handleToggle = useCallback(() => setOpen((open) => !open), []);
  const handleInnerToggle = useCallback(
    () => setInnerOpen((open) => !open),
    [],
  );

  return (
    <div style={{height: '200px'}}>
      <Card sectioned>
        <Stack vertical>
          <Button
            onClick={handleToggle}
            ariaExpanded={open}
            ariaControls="basic-collapsible"
          >
            Toggle
          </Button>
          <Button
            onClick={handleInnerToggle}
            ariaExpanded={innerOpen}
            ariaControls="basic-collapsible"
          >
            Toggle inner
          </Button>
          <Collapsible
            open={open}
            id="basic-collapsible"
            transition={{duration: '500ms', timingFunction: 'ease'}}
            expandOnPrint
          >
            <TextContainer>
              Your mailing list lets you contact customers or visitors who have
              shown an interest in your store. Reach out to them with exclusive
              offers or updates about your products.
            </TextContainer>
            <Collapsible
              open={innerOpen}
              id="basic-inner-collapsible"
              transition={{duration: '500ms', timingFunction: 'ease-out'}}
              expandOnPrint
            >
              <TextContainer>
                Your mailing list lets you contact customers or visitors who
                have shown an interest in your store. Reach out to them with
                exclusive offers or updates about your products.
              </TextContainer>
            </Collapsible>
          </Collapsible>
        </Stack>
      </Card>
    </div>
  );
}
```

Reply:
@AndrewMusgrave this works with this PR, am I missing something?